php/driftphp/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "drift/skeleton",
+    "description": "Skeleton repository ofr DriftPHP",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marc Morera",
+            "email": "yuhu@mmoreram.com"
+        }
+    ],
+    "require": {
```

Comment:
seems that their is duplicate

Reply:
I don't understand :(

Reply:
I mean that some subdependencies are listed. For example, some dependencies of https://github.com/driftphp/server/blob/master/composer.json are here, which seems to be an error

Reply:
But these are different packages. In this package we have the dependencies this package needs, and some other dependencies, which can be the same ones, are listed in the server composer file because the server, in an isolated way, needs them.

Reply:
seems this could lead to version mismatch for `driftphp` ... but anyone we could consider it at resolved